src/groups/mqb/mqbstat/mqbstat_queuestats.cpp

```diff
@@ -553,6 +549,38 @@ void QueueStatsDomain::reportConfirmTime(bsls::Types::Int64 value,
     }
 }
 
+void QueueStatsDomain::reportQueueTime(bsls::Types::Int64 value,
+                                       const bsl::string& appId)
+{
+    BSLS_ASSERT_SAFE(d_statContext_mp && "initialize was not called");
+
+    // If there is no subcontexts, report `queue time` metric to the queue
+    // context
+    if (!d_subContexts_mp) {
+        return d_statContext_mp->reportValue(
+            DomainQueueStats::e_STAT_QUEUE_TIME,
+            value);  // RETURN
```

Comment:
Is returning another void function intended here? Althow it's legal, it looks confusinfg. Consider rewrite it like this:
```
d_statContext_mp->reportValue(
            DomainQueueStats::e_STAT_QUEUE_TIME,
            value);
return;  // RETURN
```